src/screens/FeedbackScreen.js

```diff
@@ -1,141 +1,183 @@
+              />
+
+              <Input
+                name="email"
+                label={texts.feedbackScreen.inputsLabel.email + ' *'}
+                boldLabel
+                placeholder={texts.feedbackScreen.inputsLabel.email}
+                keyboardType="email-address"
+                validate
+                rules={{
+                  required: texts.feedbackScreen.inputsErrorMessages.email,
+                  pattern: {
+                    value: EMAIL_REGEX,
+                    message: texts.feedbackScreen.inputsErrorMessages.email
+                  }
+                }}
+                errorMessage={errors.email && errors.email.message}
+                control={control}
+                containerStyle={styles.containerStyle}
+              />
+
+              <Input
+                name="phone"
+                label={texts.feedbackScreen.inputsLabel.phone + ' *'}
+                boldLabel
+                placeholder={texts.feedbackScreen.inputsLabel.phone}
+                keyboardType="phone-pad"
+                validate
+                rules={{ required: texts.feedbackScreen.inputsErrorMessages.phone }}
+                errorMessage={errors.phone && errors.phone.message}
+                control={control}
+                containerStyle={styles.containerStyle}
+              />
+
+              <Input
+                name="message"
+                label={texts.feedbackScreen.inputsLabel.message + ' *'}
+                boldLabel
+                placeholder={texts.feedbackScreen.inputsLabel.message}
+                keyboardType="phone-pad"
+                multiline
+                textAlignVertical="top"
+                validate
+                rules={{ required: texts.feedbackScreen.inputsErrorMessages.message }}
+                errorMessage={errors.message && errors.message.message}
+                control={control}
+                containerStyle={styles.containerStyle}
+                inputStyle={styles.textArea}
+              />
+
+              <Controller
+                name="consent"
+                rules={{ required: texts.feedbackScreen.inputsErrorMessages.checkbox }}
+                render={({ onChange, value }) => (
+                  <>
+                    <Checkbox
+                      boldTitle
+                      title={texts.feedbackScreen.inputsLabel.checkbox + ' *'}
+                      checkedIcon="check-square-o"
+                      checkedColor={colors.accent}
+                      uncheckedIcon="square-o"
+                      uncheckedColor={colors.darkText}
+                      checked={value}
+                      onPress={() => onChange(!value)}
+                    />
+                    <Input
+                      control={control}
+                      errorMessage={errors.consent && errors.consent.message}
+                      hidden
+                      name={'consent'}
+                      validate
+                    />
```

Comment:
please adjust the error text for the checkbox with less padding from the checkbox and label

<img width="373" alt="image" src="https://user-images.githubusercontent.com/1942953/232495359-711a6f9a-31ae-4343-a713-d5bb09853365.png">

there should be more padding to the button compared to the checkbox and label

Reply:
the problem is that the input is hidden. even though it is hidden, a space is automatically created for the `label`.
solving the problem.

https://github.com/smart-village-solutions/smart-village-app-app/pull/741/commits/b8634cc9c32965eae9f7f95d7dff459a2e310ed9